packages/prisma-client/prisma/schema.prisma

```diff
@@ -135,18 +135,19 @@ view UserProduct {
 }
 
 model Project {
-  id                 String                 @id @default(uuid())
-  createdAt          DateTime               @default(now())
-  title              String
-  domain             String                 @unique
-  user               User?                  @relation(fields: [userId], references: [id])
-  userId             String?
-  build              Build[]
-  isDeleted          Boolean                @default(false)
-  files              File[]
-  projectDomain      ProjectDomain[]
-  latestBuild        LatestBuildPerProject?
-  authorizationToken AuthorizationToken[]
+  id                  String                 @id @default(uuid())
+  createdAt           DateTime               @default(now())
+  title               String
+  domain              String                 @unique
+  user                User?                  @relation(fields: [userId], references: [id])
+  userId              String?
+  build               Build[]
+  isDeleted           Boolean                @default(false)
+  files               File[]
+  projectDomain       ProjectDomain[]
+  latestBuild         LatestBuildPerProject?
+  authorizationToken  AuthorizationToken[]
+  previewImageAssetId String
```

Comment:
its not string but relation to assets table, with relation you will not even need to left join view